tools/test.sh

```diff
@@ -19,9 +19,28 @@ function prepare {
   # Webpack with the proto loader isn't used when running the tests, so the proto files need to be prepared manually
   pbjs -t json "webknossos-datastore/proto/SkeletonTracing.proto" > "$testBundlePath/SkeletonTracing.proto.json"
   pbjs -t json "webknossos-datastore/proto/VolumeTracing.proto" > "$testBundlePath/VolumeTracing.proto.json"
-  # --copy-files will copy files that are present in the source dir but are not transpiled (e.g.: json files)
-  # "$@" inside this function refers to all arguments of the prepare function, not all arguments of this bash script
-  BABEL_ENV=test babel --extensions .ts,.tsx "$jsPath" --out-dir "$testBundlePath" --copy-files "$@"
+
+  node_modules/.bin/esbuild \
+    --platform=node \
+    --format=cjs \
+    --define:process.env.BABEL_ENV=\"test\" \
+    --target=node10.4 \
```

Comment:
In https://github.com/scalableminds/webknossos/blob/master/DEV_INSTALL.md we recommend node 16 or 18. Would it make sense to use a similar target, here? Maybe this will speed things up, because newer features can be used.

Reply:
Unfortunately, node >= 13 fails to execute some tests because of a dynamic import (which gets compiled to a `require` with lower node targets). I couldn't really find out why the import fails, but I added a comment which explains the target config.